AudioBar/AudioBar.swift

```diff
@@ -80,6 +79,21 @@ public struct AudioBar: Elm.Module {
         switch message {
 
         case .prepareToLoad(let url):
+            let isPlayerActive: Bool = {
+                switch model {
+                case .waitingForURL:
+                    return false
+                case .readyToLoadURL:
+                    return false
+                case .waitingForPlayerToBecomeReadyToPlayURL:
+                    return true
+                case .readyToPlay:
```

Comment:
Consider having `.waitingForPlayerToBecomeReadyToPlayURL` and `. readyToPlay` in one line.